tests/parsers/test.manifestjson.js

```diff
@@ -431,6 +431,13 @@ describe('ManifestJSONParser', () => {
         // unsafe-inline is not supported by Firefox and won't be for the
         // forseeable future. See http://bit.ly/2wG6LP0 for more details-
         "script-src 'self' 'unsafe-inline';",
+
+        // 'default-src' is insecure, but the limiting 'script-src' prevents
+        // remote script injection
+        "default-src *; script-src 'self'",
```

Comment:
This CSP would also allow `object-src` to fallback to `*`.

Reply:
@wagnerand out of curiosity do you know what spec of CSP web-ext's support? Or is it tied to what firefox supports for web content by default?

Reply:
Hm, yes, but we don't check for `object-src` either way.

Reply:
@muffinresearch It mostly follows the web content spec, the specifics (and differences) are documented at https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/content_security_policy

Reply:
@muffinresearch We discussed this with Security (I can't find the issue right now), and came to the conclusion that only `script-src` and `worker-src` are relevant to catch a CSP that allows remote script injection.

Reply:
This was implemented in https://github.com/mozilla/addons-linter/issues/1298 but I might have that `object-src` information from IRC, I can't find it unfortunately.

Reply:
> @muffinresearch We discussed this with Security (I can't find the issue right now), and came to the conclusion that only script-src and worker-src are relevant to catch a CSP that allows remote script injection. Do you have info that suggests otherwise?

There's presumably a reason the default web-ext CSP sets `object-src 'self'` to prevent remote objects by default?

Reply:
This seems out of scope for this PR. Can we file a new issue, please?

Reply:
@wagnerand @EnTeQuAk also FWIW - not setting object-src is called out here as explicitly invalid: https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/content_security_policy#Invalid_examples